ros_ws/src/common/constants.h

```diff
@@ -22,15 +38,27 @@ const int X_AXIS_R_STICK = 2;
  */
 const std::string JOY_TOPIC = "joy";
 /**
- * @brief The topic that twist messages will be published on
+ * @brief The topic that teleop twist messages will be published on
  */
 const std::string TELEOP_TOPIC = "teleop_twist";
+/**
+ * @brief The topic that autonomous twist messages will be published on
+ */
+const std::string AUTO_TOPIC = "auto_twist";
+/**
+ * @brief The topic containing twist messages to be sent to the motor controller
+ */
+const std::string CMD_VEL_TOPIC = "cmd_vel";
```

Comment:
Let's rename CMD_VEL_TOPIC to be something more generic so we wouldn't be locked into using the particular topic called "cmd_vel". Maybe CONTROL_TOPIC or something similar...